include/deal.II/grid/tria_accessor.h

```diff
@@ -2450,7 +2450,21 @@ class CellAccessor :  public TriaAccessor<dim,dim,spacedim>
   bool direction_flag () const;
 
   /**
-   * Index of the parent of this cell within the level of the triangulation to
+   * Return the how many-th active cell the current cell is (assuming
+   * the current cell is indeed active). This is useful, for example,
+   * if you are accessing the elements of a vector with as many
+   * entries as there are active cells. Such vectors are used for
+   * estimating the error on each cell of a triangulation, for
+   * specifying refinement criteria passed to the functions in
+   * GridRefinement, and for generating cell-wise output.
+   *
+   * The function throws an exception if the current cell is not
+   * active.
+   */
```

Comment:
Can you add a note that this returns the _local_ cell index in a ```distributed::Triangulation``` and includes artificial and ghost cells?